internal/output/html.go

```diff
@@ -67,15 +71,23 @@ type HTMLVulnResultSummary struct {
 	SeverityScore    string
 }
 
+// HTMLPackageDetail represents detailed layer tracing information about a package.
+type HTMLPackageDetail struct {
+	LayerCommand        string
+	LayerCommandTooltip string
+	LayerID             string
+	InBaseImage         string
```

Comment:
The InBaseImage be a boolean, and the text should be decided in the template file